aiohttp/cookiejar.py

```diff
@@ -37,6 +39,17 @@ def __init__(self, *, unsafe=False, loop=None):
         self._next_expiration = ceil(self._loop.time())
         self._expirations = {}
 
+    def load(self, file_path):
+        f = open(file_path, 'rb')
```

Comment:
Since the pickle do not support `pathlib.Path`, I choose to use `file_path = str(file_path) if isinstance(file_path, pathlib.Path) else file_path` or `file_path = str(file_path)` instead.